src/components/Tooltip/PopoverAnchorTooltip.tsx

```diff
@@ -0,0 +1,38 @@
+import {BoundsObserver} from '@react-ng/bounds-observer';
+import React, {useContext, useMemo, useRef} from 'react';
+import {PopoverContext} from '@components/PopoverProvider';
+import BaseTooltip from './BaseTooltip';
+import {TooltipExtendedProps} from './types';
+
+function PopoverAnchorTooltip({shouldRender = true, children, ...props}: TooltipExtendedProps) {
+    const {isOpen, popover} = useContext(PopoverContext);
+    const tooltipRef = useRef<BoundsObserver>(null);
+
+    const isPopoverRelatedToTooltipOpen = useMemo(() => {
+        // eslint-disable-next-line @typescript-eslint/dot-notation
+        const tooltipNode = tooltipRef.current ? tooltipRef.current['_childNode'] : null;
```

Comment:
Maybe let's do it like this? You need to check if the logic is the same though
```suggestion
        const tooltipNode = tooltipRef.current?.['_childNode'] ?? null;
```